libsharedringbuffer/include/shared_ringbuffer/shared_ringbuffer.h

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright 2022, UNSW (ABN 57 195 873 179)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <stddef.h>
+#include <sel4utils/sel4_zf_logif.h>
+#include <utils/util.h>
```

Comment:
These two includes need to have their order swapped in order to avoid the following compiler warning:
```suggestion
#include <utils/util.h>
#include <sel4utils/sel4_zf_logif.h>
```

```
[267/468] Building C object projects_libs/li...d_ringbuffer.dir/src/shared_ringbuffer.c.obj
In file included from /tmp/tmp.MkaXVLiTOr/projects/projects_libs/libsharedringbuffer/include/shared_ringbuffer/shared_ringbuffer.h:12,
                 from /tmp/tmp.MkaXVLiTOr/projects/projects_libs/libsharedringbuffer/src/shared_ringbuffer.c:7:
/tmp/tmp.MkaXVLiTOr/projects/util_libs/libutils/include/utils/util.h:40:2: warning: #warning "Attempted to set ZF_LOG_LEVEL but _ZF_LOG_LEVEL has already been defined." "Check that <utils/zf_log.h> hasn't been imported before this file, or define ZF_LOG_LEVEL explicitly before including <utils/zf_log.h>." [-Wcpp]
   40 | #warning "Attempted to set ZF_LOG_LEVEL but _ZF_LOG_LEVEL has already been defined." \
      |  ^~~~~~~
```

Reply:
Thanks!